documentation/collectionrules/collectionrules.md

```diff
@@ -46,6 +46,7 @@ The following are the currently available triggers:
 | [AspNetRequestDuration](../configuration/collection-rule-configuration.md#aspnetrequestduration-trigger) | Event Pipe | Satisfied when the number of HTTP requests have response times longer than the threshold duration. |
 | [AspNetResponseStatus](../configuration/collection-rule-configuration.md#aspnetresponsestatus-trigger) | Event Pipe | Satisfied when the number of HTTP responses that have status codes matching the pattern list is above the specified threshold. |
 | [EventCounter](../configuration/collection-rule-configuration.md#eventcounter-trigger) | Event Pipe | Satisfied when the value of a counter falls above, below, or between the described threshold. |
+| [SystemDiagnosticsMetrics](../configuration/collection-rule-configuration.md#systemdiagnosticsmetrics-trigger-71) | Event Pipe | Satisfied when the value of an instrument falls above, below, or between the described threshold. |
```

Comment:
I think this should be renamed to `Meters` or `EventMeters` to be consistent with the rest of the tool. I don't think we make any reference to SDM or System.Diagnostics.Metrics anywhere, but we do call them meters.